e2e/tests/visual/components/about.visual.spec.js

```diff
@@ -22,41 +21,33 @@
  *****************************************************************************/
 
 /*
-Collection of Visual Tests set to run with modified init scripts to inject plugins not otherwise available in the default contexts.
-
-These should only use functional expect statements to verify assumptions about the state
-in a test and not for functional verification of correctness. Visual tests are not supposed
-to "fail" on assertions. Instead, they should be used to detect changes between builds or branches.
-
-Note: Larger testsuite sizes are OK due to the setup time associated with these tests.
+Tests the branding associated with the default deployment. At least the about modal for now
 */
 
-// eslint-disable-next-line no-unused-vars
-const { test, expect } = require('../../pluginFixtures');
-const { createDomainObjectWithDefaults } = require('../../appActions');
+const { test, expect } = require('../../../pluginFixtures');
 const percySnapshot = require('@percy/playwright');
-const path = require('path');
 
-const CUSTOM_NAME = 'CUSTOM_NAME';
-
-test.describe('Visual - addInit', () => {
+test.describe('Visual - Branding', () => {
+    test.beforeEach(async ({ page }) => {
+        //Go to baseURL and Hide Tree
+        await page.goto('./#/browse/mine?hideTree=true', { waitUntil: 'networkidle' });
```

Comment:
Replace url with `./` and hide the tree by physically clicking the minimize left pane button